src/NodaTime.Web/Markdown/root/versions.md

```diff
@@ -4,6 +4,21 @@ User-visible changes from 1.0.0-beta1 onwards. See the
 [project repository](https://github.com/nodatime/nodatime) for more
 details.
 
+## 2.2.1, released 2017-10-14 with tzdb2017b
+
+Bug-fix release:
+
+- Fixes [issue 975]: `LocalInstant.SafeMinus` gives incorrect result around the start/end of time
+- Fixes [issue 971]: Clarify exception for unknown standard patterns to point to %
+- Fixes [issue 979]: Instant bounds checking bypassed
```

Comment:
"... when subtracting a `Duration`"?

(Also `Instant`, maybe? We use monospace style below.)

Reply:
SGTM